zebedee-cms/src/main/java/com/github/onsdigital/zebedee/model/approval/ApproveTask.java

```diff
@@ -363,11 +366,12 @@ protected void uploadWhitelistedFiles(Collection collection, CollectionReader co
         for (String uri : collectionReader.getReviewed().listUris()) {
             if (uri.endsWith(".csv") || uri.endsWith(".xlsx") || uri.endsWith(".xls") || uri.endsWith(".csdb")) {
                 String fileName = uri.substring(1);
-                Resource myFile = collectionReader.getResource(fileName);
+                String decodedFilename = removeEncoding(fileName);
+                Resource myFile = collectionReader.getResource(decodedFilename);
```

Comment:
Can "myFile" be renamed to something like "decodedFile" please

Reply:
This is now addressed.